pkgs/development/python-modules/expecttest/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+
+# Native build inputs
+, poetry-core
+
+# Check inputs
+, python
+, hypothesis
+}:
+
+buildPythonPackage rec {
+  pname = "expecttest";
+  version = "0.1.3";
+  format = "pyproject";
+
+  # Pypi doesn't contain the test, so fetch from GitHub
+  src = fetchFromGitHub {
+    owner = "ezyang";
+    repo = "expecttest";
+    rev = "4be7413c98b4c6bdb4b26bfc8358c0a909c8d675";
+    sha256 = "sha256:051bffhfii5vqhfpgvcl5hkbj7c24p4j93hggmhf5k92rfdcd16i";
+  };
+
+  nativeBuildInputs = [ poetry-core ];
+
+  checkInputs = [ hypothesis ];
+
+  checkPhase = "${python.interpreter} test_expecttest.py";
```

Comment:
Could you please check if the tests could be run with `pytestCheckHook`?

E.g., 

```text
  checkInputs = [
    pytestCheckHook
  ];

  pytestFlagsArray = [
    "test_expecttest.py"
  ];
```

Reply:
Done. Didn't realize I could do this, thanks